db/schema.rb

```diff
@@ -296,7 +297,7 @@
     t.string "processor_id"
     t.boolean "default"
     t.jsonb "data"
-    t.datetime "deleted_at", precision: nil
+    t.datetime "deleted_at"
```

Comment:
Let's revert this change as it is relevant to the PR.

Reply:
About those, I am wondering why was that removed since I didn't remove it manually. This was removed when I ran `rails db:migrate` when I was initializing this project. I don't know why! 

Will revert it! 👍🏽